packages/treasury/test/swingsetTests/test-treasury.js

```diff
@@ -2,6 +2,9 @@
 
 // @ts-check
 
+// TODO Remove babel-standalone preinitialization
+// https://github.com/endojs/endo/issues/768
+import '@agoric/babel-standalone';
```

Comment:
I suspect this should be using `prepare-test-env-ava.js`. @Chris-Hibbert ? @katelynsills ?

Reply:
That sounds right, but I don't see any consistency in other swingset tests to be able to tell what the model should be. 

Ah, I see `prepare-test-env-ava.js` is  a replacement for several lines here that setup `rawtest`, `test`, etc.